cura/CuraApplication.py

```diff
@@ -974,6 +882,7 @@ def registerObjects(self, engine):
         qmlRegisterType(MachineNameValidator, "Cura", 1, 0, "MachineNameValidator")
         qmlRegisterType(UserChangesModel, "Cura", 1, 0, "UserChangesModel")
         qmlRegisterSingletonType(ContainerManager, "Cura", 1, 0, "ContainerManager", ContainerManager.createContainerManager)
+        qmlRegisterSingletonType(SettingVisibilityPresetsModel, "Cura", 1, 0, "SettingVisibilityPresetsModel", SettingVisibilityPresetsModel.createSettingVisibilityPresetsModel)
```

Comment:
@fieldOfView Just a note that we are reaching the limited number of singletons QML allows us to use, which is 30. We need to be careful not to add more singletons to QML...

Reply:
Should we start using some sort of UM.Models model?

Reply:
Whut, there is a limit to that?!

Models like these don't need to be singletons per-se of course.